node-js-basics-101/node_modules/ws/lib/Constants.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const safeBuffer = require('safe-buffer');
+
+const Buffer = safeBuffer.Buffer;
+
+exports.BINARY_TYPES = ['nodebuffer', 'arraybuffer', 'fragments'];
```

Comment:
__[jshint]__
* 'exports' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->